### DISCUSSION TITLE
Abort Fil+ program by removing restriction to access to multiplier and allowing unrestricted minting of datacap?

### DISCUSSION THREAD

**Discussion by brian75235** [filecoin-project/FIPs#1082] [Polls]
The Fil+ program is a pain in the neck for all SPs. And it is highly centralized, awfully inefficient, and not really achieving its aim. 
1. Highly centralized, basically, Galen and Kevin decide who and how much DC people get, and have no clear guidelines and rule for this decision, which can be very subjective (I am not saying their decisions to be wrong, but mechanically not right). It is even worse than 1-2 years ago, before so called allocator pathways. 
2.  Awfully inefficient, takes forever to review and get the DC. SPs who are actively sealing take too long time waiting and dealing with DC allocations. The SPARK system is almost the only data source they rely on, which has its own defections and been complaint by a lot of clients and SPs (well, it is a useful tool, but every tool has its defection), and the Fil+ team claims they are too busy to develop new tools or improve the current tools. I really can not understand this. 
3. Not really achieving its aim. Actually, even with soooo much trouble, still, 99.9% of the onboarded data are still garbage, so why bother? It is like spending so much time and energy to decorate a piece of shit. 

With above, I would like to re-mention this FIP: https://github.com/filecoin-project/FIPs/discussions/844, by starting this poll. 

Fil+ 项目对所有 SP（存储提供商）来说都是个大麻烦。它高度集中、效率极其低下，并且并未真正实现其目标。

1. 高度集中化：基本上，Galen 和 Kevin 决定谁能获得 DC（数据容量）以及获得多少，但没有明确的指导方针和规则来支持这些决策，这使得这些决策可能带有很强的主观性。（我不是说他们的决策一定是错的，而是决策机制本身不合理。）相比 1-2 年前所谓的分配路径（allocator pathways）出现之前，现在的情况甚至更糟。

2. 效率极低：审核和分配 DC 需要花费非常长的时间。那些积极封装数据的 SP 需要花费大量时间等待和处理 DC 分配问题。他们几乎完全依赖 SPARK 系统作为数据来源，而这个系统本身就存在很多缺陷，已经受到许多客户和 SP 的抱怨。（虽然 SPARK 是一个有用的工具，但任何工具都有其缺陷。）而 Fil+ 团队却声称他们太忙了，无法开发新工具或改进现有工具。这一点我完全无法理解。

3. 未能真正实现其目标：实际上，即使经过这么多麻烦，仍然有 99.9% 的上链数据是垃圾数据，那又何必费劲？这就像花费大量时间和精力去装饰一坨屎。

基于以上几点，我想再次提及这个 FIP：[https://github.com/filecoin-project/FIPs/discussions/844，并发起这项投票。](https://github.com/filecoin-project/FIPs/discussions/844%EF%BC%8C%E5%B9%B6%E5%8F%91%E8%B5%B7%E8%BF%99%E9%A1%B9%E6%8A%95%E7%A5%A8%E3%80%82)

**Comment by maybeLn** [comment]
至从fil+出现后，sp陆续离开。这是傻逼决策，看似去中心化，实际上还是fil开发团队说了算。fil+的出现币价一直跌，完全起不来，什么降低sp加入的成本，结果呢？币价低，成本低，sp数量新增了吗？一个连sp都看不到希望的项目，指望散户看懂？为何btc的机器贵的同时全球各地还在增加节点？fil是反其道而行之，节点在不停的到期退出。

**Comment by Derek-zd** [comment]
他们只想赚钱，你却想让FIL变得更好。闹呢

**Reply by Fatman13** [reply]
MFGA! Make Filecoin Great Again! 😂

**Comment by de-authority** [comment]
Yeah. You're right. This is a stupid system that should have been shut down. No matter what consequences we face, I  voted yes!

**Comment by z12crypto** [comment]
支持

**Comment by likuo5211-cmyk** [comment]
2025年11.11日了，依旧不做出改变的垃圾项目

**Comment by haheihou5** [comment]
兄弟，不如推一下 https://github.com/filecoin-project/FIPs/discussions/1176 比较现实，这个和大部分关于fil+的提议类似，更有可行性

**Comment by haheihou5** [comment]
之前的提议都比较过时了，我明天会发布一个新的，系统性的提议，希望你能来支持。我也不推#1176了因为我实在看不明白
